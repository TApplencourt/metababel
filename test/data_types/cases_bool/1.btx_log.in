event: { pf_1 = true, pf_2 = false }
