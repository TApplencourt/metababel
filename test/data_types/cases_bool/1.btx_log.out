Trace class:
  Stream class `sc` (ID 0):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event` (ID 0):
      Payload field class: Structure (2 members):
        pf_1: Boolean
        pf_2: Boolean

{Trace 0, Stream class ID 0, Stream ID 0}
Stream beginning:
  Class name: sc
  Trace:
    Stream (ID 0, Class ID 0)

Trace class:
  Stream class `sc` (ID 0):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event` (ID 0):
      Payload field class: Structure (2 members):
        pf_1: Boolean
        pf_2: Boolean

{Trace 1, Stream class ID 0, Stream ID 0}
Stream beginning:
  Class name: sc
  Trace:
    Stream (ID 0, Class ID 0)

{Trace 0, Stream class ID 0, Stream ID 0}
Event `event` (Class ID 0):
  Payload:
    pf_1: Yes
    pf_2: No

{Trace 1, Stream class ID 0, Stream ID 0}
Stream end

{Trace 0, Stream class ID 0, Stream ID 0}
Stream end
